pubspec.lock

```diff
@@ -21,21 +21,21 @@ packages:
       name: archive
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.0.13"
+    version: "2.0.11"
```

Comment:
Just curious - were these changed automatically?

Reply:
Yes this did change automatically when I got the dependencies